app/components/institution-page/campus-program-application-modal/enter-email-step-container.ts

```diff
@@ -13,6 +13,8 @@ interface Signature {
 
   Args: {
     institution: InstitutionModel;
+    prefilledEmail?: string;
+    onSubmit?: () => void;
```

Comment:
Doesn't need to be optional, we have exactly one usage of this component and the value is always passed in there